### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by szarnyasg**

### CODE REVIEW

**Comment by ArnauPrat** [comment]
@szarnyasg How can I compile this such that I see the changes in the pdf? I am running "make all" but that changes inthe .yml files do not appear in the .pdf version of the document. What am I doing wrong?

**Comment by szarnyasg** [comment]
Run `./generate-tex.py`. I deliberately did not commit the generated TeX files this time to reduce the noise of this PR.

File: query-specifications/bi-read-09.yaml [hunk 1]
```diff
@@ -19,7 +24,7 @@ result:
     type: 64-bit Integer
   - name: count1
     type: 32-bit Integer
-    description: Number of Posts with at least one tag belonging to tagClass1
+    description: Number of `post1*Posts* with at least one g belonging to tagClass1
```
Comment (ArnauPrat) [inline query-specifications/bi-read-09.yaml]: Please review this sentence, it is wrong

File: query-specifications/bi-read-19.yaml [hunk 2]
```diff
@@ -5,9 +5,13 @@ title: Stranger's interaction
 description: |
   For all the *Persons* born after a certain `date`, find all the strangers they interacted with, where strangers are *Persons* that do not *know* each other. There is no restriction on the date that strangers were born.
 
-  Consider only strangers that are members of *Forums* tagged with a *Tag* of `tagClass1` (direct children not transitive) AND members of *Forums* tagged with a *Tag* of `tagClass2` (direct children not transitive). The tags may be attached to the same *Forum* or they may be attached to different *Forums*.
+  Consider only strangers that are
```
Comment (ArnauPrat) [inline query-specifications/bi-read-19.yaml]: This card is not being rendered well on the PDF. The bullet points do not appear

**Comment by szarnyasg** [comment]
@ArnauPrat good job spotting these! I fixed both issues.

**Pull request merged**